schemas/context/profilestitchidentity.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+    "meta:license": [
+        "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$id": "https://ns.adobe.com/xdm/context/profileStitchIdentity",
+    "$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
also needs `@id`

Reply:
see earlier comment on it, we will pick up all the `@id`s in a dedicated PR.